model_analyzer/config/generate/quick_plus_concurrency_sweep_run_config_generator.py

```diff
@@ -65,6 +66,8 @@ def __init__(self, search_config: SearchConfig,
             List of models to profile
         ensemble_composing_models: Dict of List of ModelProfileSpec
             Dict indexed by model name of list of composing models to profile
+        bls_composing_models: List of ModelProfileSpec
```

Comment:
Why is the format for bls vs ensemble so different?

Reply:
Because you can have multiple models, one (or more) of which is an ensemble. But, it doesn't work that way for BLS since we need to specify composing models using the config, so as of now we'll only support a single BLS when running MA.